draft-ietf-taps-impl.md

```diff
@@ -327,31 +327,29 @@ Branching for derived endpoints is the final step, and may have multiple layers
 
 Implementations should sort the branches of the tree of connection options in order of their preference rank. 
 Leaf nodes on branches with higher rankings represent connection attempts that will be raced first.
-Implementations should order the branches to reflect the preferences expressed by the application for its new connection, including Protocol and Path Selection Properties, which are specified in {{I-D.ietf-taps-interface}}. 
+Implementations should order the branches to reflect the preferences expressed by the application for its new connection, including Selection and Connection Properties, which are specified in {{I-D.ietf-taps-interface}}. 
 
 In addition to the properties provided by the application, an implementation may include additional criteria such as cached performance estimates, see {{performance-caches}}, or system policy, see {{role-of-system-policy}}, in the ranking.
-Two examples of how the Protocol and Path Selection Properties may be used to sort branches are provided below:
+Two examples of how the Selection and Connection Properties may be used to sort branches are provided below:
 
```

Comment:
Connection properties should not be used for this.

Reply:
I just made a commit that updates all other occurrences of "Selection and Connection", but I left this one standing on purpose: I think, because Connection Properties CAN be used to guide the decisions later, it is useful to have them in the example here as a way to remind the reader of this possibility.